packages/esm-patient-banner-app/src/banner/patient-banner.scss

```diff
@@ -1,6 +1,8 @@
+@use '@carbon/styles/scss/spacing';
 @use '@carbon/styles/scss/type';
 @import '~@openmrs/esm-styleguide/src/vars';
 
+
 .container {
   border-top: 0.063rem solid $ui-03;
```

Comment:
Why not use carbon spacing

Reply:
Counterpoint: I actually reckon we shouldn't be using the Carbon spacing tokens - they're not parseable at a glance in my opinion and I reckon we should prefer using em/rem-based spacing directly.